### PR TITLE
feat: add optout models for echo19

### PR DESCRIPTION
**Pull request by jitoquinto**
https://techallies.atlassian.net/browse/PAD-2900

echo19 has new set of data they are sending our way. Our partners would like this data so this PR adds the dbt models so we can start delivering the optouts table to them.

Dev Testing - https://f9ebbc83f6f3414381d6fa9a52957253-dot-us-central1.composer.googleusercontent.com/dags/echo19/grid?tab=details&dag_run_id=scheduled__2024-10-22T08%3A45%3A00%2B00%3A00

### CODE REVIEW

File: dbt-cta/echo19/models/1_cta_incremental/_1_cta_incremental__models.yml [hunk 1]
```diff
@@ -207,4 +207,12 @@ models:
         tests:
           - unique
           - not_null
+      - name: optouts_base
+        description: ''
+        columns:
+          - name: optOutId
```
Comment (emily-flambe) [inline dbt-cta/echo19/models/1_cta_incremental/_1_cta_incremental__models.yml:213]: yay

File: dbt-cta/echo19/models/1_cta_incremental/optouts_base.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{{ config(
+    unique_key = "optOutId"
```
Comment (emily-flambe) [inline dbt-cta/echo19/models/1_cta_incremental/optouts_base.sql:2]: yay

**Review by emily-flambe — APPROVED** [review]
lgtm!

**Review by kanelouise — APPROVED** [review]
looks good to me, too! 

not a blocker AT ALL, just a heads up: the sync seems to be running in dev still (probably bc of the sync start date).

**Pull request merged**